projects/pgai/db/sql/idempotent/014-semantic-catalog.sql

```diff
@@ -0,0 +1,1004 @@
+
```

Comment:
do we need function to grant permissions on a semantic catalog? How about grant ability to create/drop semantic catalogs?

Reply:
Do we have any dump/restore tests?